apps/heft/src/plugins/TypeScriptPlugin/TypeScriptBuilder.ts

```diff
@@ -668,41 +666,43 @@ export class TypeScriptBuilder extends SubprocessRunnerBase<ITypeScriptBuilderCo
     ts: ExtendedTypeScript,
     tsconfig: TTypescript.ParsedCommandLine,
     genericProgram: TTypescript.BuilderProgram | TTypescript.Program,
-    writeFile: TTypescript.WriteFileCallback
+    filesToWrite: IFileToWrite[]
```

Comment:
It might be cleaner to include this in the return value instead.

Reply:
Fixed